xarray/core/dataarray.py

```diff
@@ -6670,6 +6674,72 @@ def resample(
             **indexer_kwargs,
         )
 
+    def to_dask_dataframe(
+        self,
+        dim_order: Sequence[Hashable] | None = None,
+        set_index: bool = False,
+        *,
+        name: Hashable | None = None,
```

Comment:
```suggestion
```

Reply:
I have made the changes .Please review them.